gatsby-ssr.js

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react'
+import { PreviewStoreProvider } from 'gatsby-source-prismic'
+
+export const wrapRootElement = ({ element }) => (
+  <PreviewStoreProvider>{element}</PreviewStoreProvider>
+)
```

Comment:
Can someone explain what this is and why it's identical to gatsby-browser.js?

Reply:
The file is for server side rendering.
When using the wrapRootElement Gatsby recommend using both the ssr and browser api together. https://www.gatsbyjs.com/docs/ssr-apis/#wrapRootElement

`PreviewStoreProvider` providers a store for preview data.

Anymore questions?